bson/codec_options.py

```diff
@@ -35,7 +36,7 @@ def _raw_document_class(document_class):
 _options_base = namedtuple(
     'CodecOptions',
     ('document_class', 'tz_aware', 'uuid_representation',
-     'unicode_decode_error_handler', 'tzinfo'))
+     'unicode_decode_error_handler', 'tzinfo', 'encoder_map', 'decoder_map'))
```

Comment:
I think we all discussed have a new class to bundle these (and future related options)?